tests/Visual/SingleTestOrDirectory.php

```diff
@@ -3,63 +3,54 @@
 use Symfony\Component\Process\Process;
 
 $run = function (string $target) {
-    $process = new Process(['./bin/pest', $target], dirname(__DIR__, 2));
+    $process = new Process(['php', 'bin/pest', $target], dirname(__DIR__, 2));
 
     $process->run();
 
     return preg_replace('#\\x1b[[][^A-Za-z]*[A-Za-z]#', '', $process->getOutput());
 };
 
 test('allows to run a single test', function () use ($run) {
-    assertStringContainsString(<<<EOF
-   PASS  Tests\Fixtures\DirectoryWithTests\ExampleTest
-  ✓ it example
-
-  Tests:  1 passed
-EOF, $run('tests/Fixtures/DirectoryWithTests/ExampleTest.php'));
+    $output = $run('tests/Fixtures/DirectoryWithTests/ExampleTest.php');
```

Comment:
Can you revert this changes? I prefer in the way it was before.

If don't work on Windows, but create a file with the contents and do a `file_get_contents` on it.

Reply:
Not sure how you mean with the file_get_contents()?
I were going to solve it differently first, by replacing the \r\n with \n, but then I understood, that there are cases, where anything that still contains linebreaks can break in more than one way and as such the line breaks should not be part of the comparison. Having a file, could still make it fail in some cases I think?

Reply:
I know, I like the other way it was also better, but this one works always.

Reply:
Can you revert all the changes in SingleTestOrDirectory.php ? Except the line 6.